tb_device_mqtt.py

```diff
@@ -66,25 +70,32 @@ def __init__(self, host, port=1883, access_token=None, quality_of_service=None,
         self._access_token = access_token
 
         if not client_id:
-            client_id = ubinascii.hexlify(machine.unique_id())
+            client_id = hexlify(unique_id())
         self._client_id = client_id
-        self._client = MQTTClient(self._client_id, self._host, self._port, self._access_token, 'pswd', keepalive=120)
+        self._client = MQTTClient(
+            self._client_id, self._host, self._port, self._access_token, 'pswd', keepalive=120
+        )
 
-    def connect(self):
+    def connect(self, timeout=5):
         try:
-            response = self._client.connect()
+            response = self._client.connect(timeout=timeout)
             self._client.set_callback(self._callback)
+
             self._client.subscribe(ATTRIBUTES_TOPIC, qos=self.quality_of_service)
             self._client.subscribe(ATTRIBUTES_TOPIC + "/response/+", qos=self.quality_of_service)
             self._client.subscribe(RPC_REQUEST_TOPIC + '+', qos=self.quality_of_service)
             self._client.subscribe(RPC_RESPONSE_TOPIC + '+', qos=self.quality_of_service)
 
             self.connected = True
-
+            print("Connected successfully!")
```

Comment:
Please remove this print, in case of needed, developer can check returned value and then print or not print, etc.